apps/Gruntfile.js

```diff
@@ -153,8 +153,16 @@ config.copy = {
       {
         expand: true,
         cwd: 'lib/droplet',
-        src: ['droplet-full' + droplet_suffix + '.js'],
-        dest: 'build/package/js/droplet/'
+        src: [
+          'droplet.js',
```

Comment:
Source maps seem like they would be useful, but I haven't thought through the right way to use them (do we want them on our production server? it looks like we need to upload the original (unminified) JS files in order for this to be useful - not sure how this works if you renamed the .min file to have the original file name).

If you have it tested and working, I'd go for it. If it doesn't work, I wouldn't bother including the .map file as it will just be confusing.